app/controllers/schools_controller.rb

```diff
@@ -44,11 +44,13 @@ def edit
   def update
     @school = School.find(params[:id])
     @school.assign_attributes(school_params)
+    debugger
```

Comment:
This needs to be removed. Not sure if this was causing tests to fail, but if not then some tests are missing.